ukol08_Burjanova/src/main/java/cz/czechitas/java/database/Clanek.java

```diff
@@ -1,59 +1,46 @@
 package cz.czechitas.java.database;
 
 public class Clanek {
-    private Long id_clanku;
```

Comment:
Please, follow Java naming conventions (no underscore in names, names with lowercase first letter)